scripts/dockerfiles/run.sh

```diff
@@ -13,18 +13,6 @@ EOL
 exit 0
 fi
 
-# Compatibility code for use with older stemcells. Move all files
```

Comment:
I think this section should become a scf -> fissile compatibility check instead.